app/views/donations/new.html.haml

```diff
@@ -1,4 +1,5 @@
 = provide :title, 'Donate'
+= provide :description, 'A monthly donation of $3 can support a student in Hack Club. Your contribution matters to our clubs.'
```

Comment:
Yeah, that'd be best.

On Mon, Apr 18, 2016 at 10:58 AM, Max Wofford <notifications@github.com>
wrote:

> In app/views/donations/new.html.haml
> <https://github.com/hackclub/website/pull/228#discussion_r60104258>:
>
> > @@ -1,4 +1,5 @@
> >  = provide :title, 'Donate'
> > += provide :description, 'A monthly donation of $3 can support a student in Hack Club. Your contribution matters to our clubs.'
>
> @zachlatta <https://github.com/zachlatta> Should I wrap these types of
> lines?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/website/pull/228/files/f7dd2a91806e194d0e871d79834559cd75900381#r60104258>
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>